pkg/manifest/manifest_test.go

```diff
@@ -720,6 +720,41 @@ func Test_include(t *testing.T) {
 				EnabledCapabilities: []configv1.ClusterVersionCapability{"cap1"},
 			},
 		},
+		{
+			name:    "multiple capabilities, 1 unknown, works",
+			profile: &defaultClusterProfile,
```

Comment:
nit: since #1343's pivot to pointer, we can leave unset all the properties that aren't the one whose filter we're exercising.